src/atoms/label/Label.css

```diff
@@ -0,0 +1,15 @@
+
+.a-label {
+  @apply block;
+  @apply mb-1;
+  @apply text-gray-600 font-normal text-base font-sans leading-relaxed;
```

Comment:
I'm not sure all properties are needed. Ca we remove text-base, font-sans and font-normal?